components/service/pocket/src/main/java/mozilla/components/service/pocket/stories/PocketRecommendationsRepository.kt

```diff
@@ -49,6 +59,32 @@ internal class PocketRecommendationsRepository(private val context: Context) {
         getAtomicStoriesFile().writeString { stories }
     }
 
+    suspend fun updateShownPocketRecommendedStories(updatedStories: List<PocketRecommendedStory>) {
```

Comment:
Now that we want to have additional properties and update some of them, we should reevaluate if the flat JSON file is still the right choice, since we do much more than "just" caching. Do you think we should use SQLite/Room?